Sources/BraintreeApplePay/BTApplePayCardNonce.m

```diff
@@ -1,4 +1,4 @@
-#import "BraintreeCoreSwiftImports.h"
+#import "Foundation/Foundation.h"
```

Comment:
I see a few new Foundation imports. Curious if these were moved over from `.m`s or why these were added?

Reply:
Similarly, seems like this was handled in other files that are now removed when the Obj-C files were deleted from core, so removing this gives us unknown type errors.